packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -87,6 +78,53 @@ on the root element).
 </div>
 ```
 
+### Maximum File Size (JavaScript)
+
+The maximum size of the uploaded file that is validated by the JavaScript plugin can be adjusted. The default value is
+10 MB. To increase the limit for example to 20 MB, add the `data-spirit-max-file-size` attribute:
+
+```html
+<div class="FileUploader" data-toggle="fileUploader">
+  <div class="FileUploaderInput" data-spirit-element="wrapper" data-spirit-max-file-size="20000000">
+    <!-- ... -->
+  </div>
+  <!-- FileUploaderList -->
+</div>
+```
+
+### Maximum number of files in queue (JavaScript)
+
+Counter of the maximum number of uploaded files. The default value is 10, but any value can be set with
+the `data-spirit-file-queue-limit` attribute:
+
+```html
+<div class="FileUploader" data-toggle="fileUploader">
+  <div class="FileUploaderInput" data-spirit-element="wrapper" data-spirit-file-queue-limit="2">
+    <!-- ... -->
+  </div>
+  <!-- FileUploaderList -->
+</div>
+```
+
+### Input behavior when the queue is filled (JavaScript)
+
+You can set the input/drop zone to be hidden or disabled when the file queue limit is reached.
+When you set `data-spirit-queue-limit-behavior` together with the desired limit for the queue.
```

Comment:
maybe we could mention here the available options? `hide` and ...?

Reply:
done